src/components/textarea/index.tsx

```diff
@@ -0,0 +1,23 @@
+import React, { ChangeEventHandler, FC, FocusEventHandler } from 'react';
+
+import { Textarea as ChakraTextarea } from '@chakra-ui/react';
+
+interface Props {
```

Comment:
Do we also want to support (or provide a default for) the [resize behaviour](https://chakra-ui.com/docs/components/textarea/usage#resize-behavior)?

Reply:
Let's check with a designer - it's fairly standard behaviour so it's worth supporting. That being said we can add it in a follow up